include/hardware.h

```diff
@@ -51,11 +52,15 @@ void OPL_AddConfigSettings(const ConfigPtr &conf);
 
 bool TANDYSOUND_GetAddress(Bitu& tsaddr, Bitu& tsirq, Bitu& tsdma);
 
-extern uint8_t adlib_commandreg;
-
 // Gravis UltraSound configuration and initialisation
 void GUS_AddConfigSection(const ConfigPtr &conf);
 
+// The Gravis UltraSound mirrors the Adlib's command register (written to base
+// address 388h) on its own port 20Ah. This function is used to pass through the
+// Adlib's value into the GUS (if available)
+void GUS_MirrorAdlibCmdPortWrite(const io_port_t port, const io_val_t reg_value,
```

Comment:
minor: AdLib is the correct casing (but `adlib` is better for the lowercase variant as `ad_lib` just looks stupid 😆)

Reply:
Will fix - thanks for catching it.